smartnews-ads-conversion-api.md

```diff
@@ -275,3 +275,71 @@ Content-Type: application/json
 | Purchase History       | PurchaseHistory                          |
 | Like                   | Like                                     |
 | Install                | Install                                  |
+
+
+## [New] Batch Request
```

Comment:
Nit: `[New]` is not sensible for API doc, but only sensible for changelog. It's more sensible to have `[since <version number>]` in API doc IMO. So the next question is, how do we version the API now? Will we mark it a new version for this change?

Reply:
I don't think we are keeping track of the version of conversion-api, correct me if I'm wrong @xyu335, so to be honest, I'm not sure how to document this in the docs. But I do agree that [New] is not appropriate.

Reply:
If not versioned, probably just use something like `[New since 2024/12/16]`